modules/uhttp/filters.go

```diff
@@ -55,8 +55,7 @@ type contextFilter struct {
 }
 
 func (f contextFilter) Apply(w http.ResponseWriter, r *http.Request, next http.Handler) {
-	ctx := ulog.NewLogContext(r.Context())
-	next.ServeHTTP(w, r.WithContext(ctx))
+	next.ServeHTTP(w, r)
```

Comment:
Need @anuptalwalkar to explain why we no longer need to wrap the request with the incoming context. If we don't, then this `contextFilter` seems like a noop and should be removed, right?